app/posts/apply-for-teacher-training/2022-07-13-improving-the-email-sent-to-candidates-about-new-interviews.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Improving the email sent to candidates about new interviews
+description: We now include the course subject and code. We also show details separately rather than in a single section.
+date: 2022-07-13
+---
+
+{% from "email/macro.njk" import appEmail %}
+
+The email sent to candidates when an interview is set up does not currently include the course name or code.
```

Comment:
Suggest changing this to the past tense for consistency with the rest of the post.

```suggestion
The email sent to candidates when an interview is set up did not include the course name or code.
```